### PR TITLE
UXD-1878 Button group align with UI kit

### PR DESCRIPTION
**Pull request by ShanAtDiligent**
### Purpose 🚀
Update Button Group font / spacing / icon to align with UI kit
https://aclgrc.atlassian.net/browse/UXD-1694

### Notes ✏️
- All the styles was updated during button update. This PR just to create variations to properly display the options

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/UXD-1878-ButtonGroup-Align-with-UI-Kit

### Screenshots �
Screen Shot 2021-12-01 at 9 32 24 AM
�

### CODE REVIEW

File: packages/ButtonGroup/stories/Example.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ButtonGroup from "../src";
+
+const Example = ({ itemData, isDisabled, ...rest }) => (
```
Comment (jamiek-acl) [inline packages/ButtonGroup/stories/Example.js:4]: we typically call it `moreProps` not `rest`, but no biggie in a story

**Review by jamiek-acl — APPROVED** [review]

File: packages/ButtonGroup/stories/ButtonGroup.stories.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
+import React from "react";
```
Comment (jamiek-acl) [inline packages/ButtonGroup/stories/ButtonGroup.stories.js:1]: This is not used; eslint error

**Review by jamiek-acl — CHANGES_REQUESTED** [review]
eslint error

**Comment by AndreyChernykh** [comment]
LGTM 
I guess we can approve it as soon as @jamiek-acl 's comments are addressed

**Review by jamiek-acl — APPROVED** [review]

**Pull request merged**